lib/streamlit/elements/button.py

```diff
@@ -147,3 +237,41 @@ def deserialize_button(ui_value: bool, widget_id: str = "") -> bool:
     def dg(self) -> "streamlit.delta_generator.DeltaGenerator":
         """Get our DeltaGenerator."""
         return cast("streamlit.delta_generator.DeltaGenerator", self)
+
+
+def marshall_file(coordinates, data, proto_download_button, mimetype, filename=None):
```

Comment:
super nit: filename should be file_name?

Reply:
filename, mimetype or file_name, mime_type - I feel like filename is pretty commonly one word..